CMakeLists.txt

```diff
@@ -22,7 +22,20 @@ include(CheckCSourceCompiles)
 include(CheckIncludeFile)
 include(CheckCCompilerFlag)
 include(CMakePackageConfigHelpers)
+include(CMakePushCheckState)
 include(GNUInstallDirs)
+include("${CMAKE_CURRENT_LIST_DIR}/cmake/sdlchecks.cmake")
```

Comment:
This isn't anywhere. (forgot to `git add` ?)

Reply:
Indeed, git equivalent of forgot to add the email attachment :)